### DISCUSSION TITLE
Add "duration of data collection"?

### DISCUSSION THREAD

**Discussion by nicebread** [RESQUE-Framework/website#101] [Ideas]
As a measure for the effort.

But: 

- Hard to verify
- Not necessarily useful, if you only assess one person per week for three years ...

**Comment by angaertner** [comment]
I see the same problems. Maybe add this to merit statement (e.g. possible criterion on which candidates can comment) if they think this is a relevant info for the commission (e.g. "data collection of rare sample in JVA took 10 years but has now extremely valuable implications"), but not as a default item in the RESQUE app.
